includes/wikia/DefaultSettings.php

```diff
@@ -1414,10 +1414,12 @@
 $wgAdDriverWadILCountries = null;
 
 /**
- * @name $wgAdDriverPreFooterAndBLBSwitchedCountries
- * List of countries to switch BLB with PreFooter on mobile
+ * @name $wgAdDriverStickyAndSwapCountries
+ * List of countries to run simultaneously two experiments:
+ * - switch BLB with PreFooter
+ * - sticky BFAB
  */
-$wgAdDriverPreFooterAndBLBSwitchedCountries = null;
+$wgAdDriverStickyAndSwapCountries = null;
```

Comment:
Basically, the same actions needed as https://github.com/Wikia/app/pull/15867/files#r208865749 Although, I guess @fraszczakszymon would tell changes in this files aren't needed at all ;-) I treat this file as documentation and when I'm adding vars to WikiFactory I copy description from here :p

Reply:
I also feel this is redundant, I'm removing it.

Reply:
Oh, well ;)